src/cli.ts

```diff
@@ -293,15 +294,32 @@ async function openPage(context: playwright.BrowserContext, url: string | undefi
   return page;
 }
 
-type Language = 'python' | 'python-async' | 'javascript';
+type Language = 'python' | 'python-async' | 'javascript' | 'csharp';
 
 async function open(options: Options, url: string | undefined, enableRecorder: boolean, language?: Language, outputFile?: string) {
   const { context, browserName, launchOptions, contextOptions } = await launchContext(options, false);
-  const outputs: CodeGeneratorOutput[] = [new TerminalOutput(process.stdout, language === 'javascript' ? 'javascript' : 'python')];
+  let terminalLanguage: string;
+  let languageGenerator: LanguageGenerator;
+
+  switch (language) {
+    case 'javascript': 
+      terminalLanguage = 'javascript'; 
+      languageGenerator = new JavaScriptLanguageGenerator();
+      break;
+    case 'csharp': 
+      terminalLanguage = 'csharp'; 
+      languageGenerator = new CSharpLanguageGenerator();
+      break;
+    default: 
```

Comment:
default feels a bit from for this use case since here it seems Python is default. But actually JavaScript is default which we defined in the CLI parameters. Maybe move it out to a small function and check implicit either with if or a switch/case statement for the values and return a generic Language Generator?

Reply:
This was based on the former line 304. It fallback to `PythonLanguageGenerator ` because the language could be `python` or `python-async`.

Reply:
That's cool, but I would also list languages explicitly and assert not reached in the default.

Reply:
Validation added. Notice that the CLI won't get there. I also added a validation message on the `program.help` call.